### DISCUSSION TITLE
How to change usage key in Supabase?

### DISCUSSION THREAD

**Discussion by reirysson** [supabase/supabase#5306] [Questions]
I'm starting to use subase, and there was a mistake and the public key ended up being posted, so I wanted to know how I can change the key

**Comment by GaryAustin1** [comment]
Hopefully you already moved on... but not really a way.  If it is the public key as long as you use RLS for protecting data not a big issue.  It is a public key after all...   
Not sure Supabase would change it for you for those reasons.

**Reply by reirysson** [reply]
Ah, I understand, but in that case, wouldn't there be some method to hide the commit that presents the key here on github?

**Reply by GaryAustin1** [reply]
I've only seen that there is no way for you to change it.  You could always try and ask support, I've not seen any comments anywhere on someone actually doing that and if it worked out.  I did see a discussion that signups have to have to involve your URL so at least people can not just randomly signup to your site.   Anyone can see your public key if they really want to on an app, but I know I would not be thrilled with it floating about either.

Edit:  I don't know much about github repositories, but I've seen lots of code where the actually keys are in a setup or environment file separate from the actual code.  Sorry can't help more on that.

**Reply by reirysson** [reply]
Ah, ok friend, I understand, thanks for your help, about the file with the keys, I managed to do that, but it was after I made the commit with my KEY and URL.
